targets/extension

```diff
@@ -35,12 +41,14 @@ if [ "$XMETHOD" = 'xorg' ] || grep -q xorg /etc/crouton/targets; then
 fi
 
 # Use croutonurlhandler as a fallback URL handler
-handler='/usr/local/bin/croutonurlhandler'
-for link in x-www-browser gnome-www-browser www-browser; do
-    if ! update-alternatives --query "$link" | grep -q "$handler"; then
-        update-alternatives --install "/usr/bin/$link" "$link" "$handler" 15
-    fi
-done
+if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
+    handler='/usr/local/bin/croutonurlhandler'
+    for link in x-www-browser gnome-www-browser www-browser; do
+        if ! update-alternatives --query "$link" | grep -q "$handler"; then
+            update-alternatives --install "/usr/bin/$link" "$link" "$handler" 15
```

Comment:
Here's what the Gentoo docs say about setting default applications: https://wiki.gentoo.org/wiki/DefaultApplication

To the best of my knowledge, packages will install freedesktop compatible MIME entries for default applications, so things like Chrome should "just work" providing that the DE supports it. I'll look into duplicating this for the crouton url handler.